provider/kubernetes/kubernetes.go

```diff
@@ -219,8 +220,13 @@ func (p *Provider) loadIngresses(k8sClient Client) (*types.Configuration, error)
 					}
 				}
 
-				if rule := getRuleForPath(pa, i); rule != "" {
-					templateObjects.Frontends[baseName].Routes[pa.Path] = types.Route{
+				rule, err := getRuleForPath(pa, i)
+				if err != nil {
+					log.Errorf("fail to get rule from ingress path: %s", err)
+					continue NEXT_INGRESS
```

Comment:
I think you need to invoke `delete(templateObjects.Frontends, r.Host+pa.Path)` before skipping.